spec/features/dashboard_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
+
+feature 'dashboard' do
+  context 'authorized user' do
+    scenario 'can see other users successfully' do
+      user1 = FactoryGirl.create(:user)
+      user2 = FactoryGirl.create(:user)
+      sign_in_as(user2)
+      visit root_path
+
+      expect(page).to have_content(user1.email)
+      expect(page).to have_content(user2.email)
+    end
+
+    scenario 'can filter for looking to pair' do
+      user = FactoryGirl.create(:user)
+      pair1 = FactoryGirl.create(:status_pair)
+      pair2 = FactoryGirl.create(:status_pair)
+      zone = FactoryGirl.create(:status_zone)
+
+      sign_in_as(user)
+      visit dashboard_path
+
+      click_on 'All Ready To Pair'
+      expect(page).to have_content(pair1.user.email)
+      expect(page).to have_content(pair2.user.email)
+      expect(page).to_not have_content(zone.user.email)
+    end
+
+    scenario 'can filter for open to help' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.